myapp/spec/system/tasks_spec.rb

```diff
@@ -0,0 +1,527 @@
+require 'rails_helper'
+
+describe 'Tasks', type: :system do
+  include TaskHelper
+
+  describe '1 #index' do
+
+    let(:task_one) { FactoryBot.create(:task_not_started) }
+    let(:task_two) { FactoryBot.create(:task_not_started) }
+
+    context '1-1 タスク1件' do
+
+      let!(:task_index_1) { task_one }
+
+      it '1-1-1 タイトルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_1.title
+      end
+
+      it '1-1-2 ラベルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_1.label
+      end
+
+      it '1-1-3 作成ボタンを押下することでタスク作成画面へ遷移すること' do
+        visit root_path
+        click_on '作成'
+        expect(page).to have_current_path new_task_path
+      end
+
+      it '1-1-4 詳細ボタンを押下することでタスク詳細画面へ遷移すること' do
+        visit root_path
+        click_on '詳細', match: :first
+        expect(page).to have_current_path task_path(task_index_1)
+      end
+
+    end
+
+    context '1-2 タスク複数件' do
+
+      let!(:task_index_2_one) { task_one }
+      let!(:task_index_2_two) { task_two }
+
+      it '1-2-1 1件目 タイトルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_2_one.title
+      end
+
+      it '1-2-2 1件目 ラベルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_2_one.label
+      end
+
+      it '1-2-3 2件目 タイトルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_2_two.title
+      end
+
+      it '1-2-4 2件目 ラベルが一致すること' do
+        visit root_path
+        expect(page).to have_content task_index_2_two.label
+      end
+
+      it '1-2-5 作成ボタンを押下することでタスク作成画面へ遷移すること' do
+        visit root_path
+        click_on '作成'
+        expect(page).to have_current_path new_task_path
+      end
+
+      it '1-2-6 1件目 詳細ボタンを押下することでタスク詳細画面へ遷移すること' do
+        visit root_path
+        all('a', :text => '詳細')[0].click
+        expect(page).to have_current_path task_path(task_index_2_two)
+      end
+
+      it '1-2-7 2件目 詳細ボタンを押下することでタスク詳細画面へ遷移すること' do
+        visit root_path
+        all('a', :text => '詳細')[1].click
+        expect(page).to have_current_path task_path(task_index_2_one)
+      end
+
+    end
+
+  end
+
+  describe '2 #new' do
+
+    context '2-1 全項目入力' do
+
+      let(:new_task) {
+        {
+          title: '新規タスク 全項目入力 タイトル',
+          content: '新規タスク 全項目入力 内容',
+          label: '新規タスク 全項目入力 ラベル',
+        }
+      }
+
+      it '2-1-1 登録内容が想定通りであること' do
+
+        # 画面遷移
+        visit new_task_path
+
+        # 新規タスク登録
+        fill_in 'task[title]', with: new_task[:title]
+        fill_in 'task[content]', with: new_task[:content]
+        fill_in 'task[label]', with: new_task[:label]
+        # ボタン押下
+        click_on '作成'
+
+        # 項目比較
+        expect(Task.find_by(title: new_task[:title], content: new_task[:content], label: new_task[:label])).not_to be_nil 
```

Comment:
せっかくnet_taskをハッシュで定義しているので、以下のように記載すると良いでしょう。
また、nilではないこと、だと二重否定なので、存在すること、とした方が直感的にわかりやすいです。
```suggestion
        expect(Task.find_by(new_task)).to be_present
```

あと編集してわかったのですが、この行の最後に半角スペースが残っています。
vscodeの設定で、保存時に行の最後にある無駄なスペースをトリムできるので、設定してみてください。
ただし、設定ファイルはpushしないようお願いします。

Reply:
二重否定を修正しました。
> あと編集してわかったのですが、この行の最後に半角スペースが残っています。

こちら失礼しました。
設定したつもりができておらず、、。再度設定し、保存のタイミングで末尾の半角スペースがトリムされるよう修正しました。